packages/peregrine/lib/talons/CheckoutPage/__tests__/__snapshots__/useCheckoutPage.spec.js.snap

```diff
@@ -29,6 +29,12 @@ Object {
   "setPaymentInformationDone": [Function],
   "setShippingInformationDone": [Function],
   "setShippingMethodDone": [Function],
+  "shippingInformationRef": Object {
+    "current": undefined,
```

Comment:
These are expected because these are refs that haven't been assigned to any components.